components/Step5/Step5.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import {Text, View} from 'react-native';
+import {wizard} from '../../utils/constants';
+import {selectWizard} from '../../redux/reducers/WizardData';
+import {useSelector} from 'react-redux';
+import {fitnessWizardSummary} from '../../utils/fitnessConstants';
+
+import globalStyle from '../../assets/styles/globalStyle';
+import style from './style';
+
+const Step5 = () => {
+  const wizardData = useSelector(selectWizard);
+
+  return (
+    <View style={style.summaryContainer}>
+      <View style={[style.accountDetails, style.summaryBlock]}>
+        <View style={globalStyle.marginRight20}>
+          {fitnessWizardSummary.map((summary, index) => (
+            <Text
+              key={index + summary.label}
+              style={[
+                globalStyle.FontPlayfairDisplay,
+                globalStyle.MSize,
+                globalStyle.bolderWeight,
+                style.summaryText,
+              ]}>
+              {summary.label}
+            </Text>
+          ))}
+        </View>
+
+        <View>
+          {fitnessWizardSummary.map((summary, index) => (
```

Comment:
You are iterating this array twice, this automatically means that these two should be in 1 component and the array needs to be iterated only once, otherwise the complexity of the array becomes Array.length * 2 which can be problematic in other situations.